src/pip/_internal/wheel.py

```diff
@@ -163,13 +163,15 @@ def message_about_scripts_not_on_PATH(scripts):
         grouped_by_dir[parent_dir].add(script_name)
 
     # We don't want to warn for directories that are on PATH.
-    not_warn_dirs = os.environ["PATH"].split(os.pathsep)
+    not_warn_dirs = list(
```

Comment:
Style nit - why use `list(...)` here rather than a comprehension `[...]`? Just curious, it's not that it matters.

Reply:
Ah. That's just me being lazy. Will fix.